sensu/plugins/check-large-receive-offload.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+#
+# Checks that large receive offload is disabled
+#
+# Return CRITICAL if large receive offload is enabled
+#
+# Dean Daskalantonakis <ddaskal@us.ibm.com>
+
+import subprocess
+import argparse
+import sys
+import re
+
+STATE_OK = 0
+STATE_WARNING = 1
+STATE_CRITICAL = 2
+CRITICALITY = 'critical'
+
+def switch_on_criticality():
+    if CRITICALITY == 'warning':
+        sys.exit(STATE_WARNING)
+    else:
+        sys.exit(STATE_CRITICAL)
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-i', '--interface', help='primary interface')
+    parser.add_argument('-z', '--criticality', default='critical')
```

Comment:
I've cleaned up the code so that any input other than 'critical' will be treated as warning